e2e/tests/uniswap_trade_test.rs

```diff
@@ -181,14 +182,14 @@ async fn test_with_ganache() {
         .call()
         .await
         .expect("Couldn't fetch TokenB's balance");
-    assert_eq!(balance, to_wei(80));
+    assert_eq!(balance, U256::from(99650498453042316810u128));
```

Comment:
ubernit: Maybe include `_` separators in the numerical literal so its easy to read:
```suggestion
    assert_eq!(balance, U256::from(99_650_498_453_042_316_810u128));
```

You can even enforce this with clippy with the `#![deny(clippy::unreadable_literal)]` https://rust-lang.github.io/rust-clippy/master/#unreadable_literal